docker/Dockerfile.ort-torch181-onnxruntime-nightly-cu102-cudnn7-devel-ubuntu18.04

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License.
 
 # CUDA development image for building sources
-FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04 as builder
+FROM nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04 as builder
 
 ARG TORCH_CUDA_VERSION=cu102
 ARG TORCH_VERSION=1.8.1
```

Comment:
Is this intentional? Is there any usage for torch 1,81 on our end?

Reply:
This is intentional in the sense that this was how we initially designed the pipelines. There is work that we need to do to update the torch version and simplify the packaging process. As DRI this week, I wanted to first fix the pipeline.